frontend/src/stores/roles/actions.ts

```diff
@@ -1,25 +1,38 @@
-import { get, post } from '@/services/http'
+import { get, post, put } from '@/services/http'
 import { getRoles } from './getters'
-import { roles } from './state'
+import { setRoles, roles } from './state'
 import type { Role } from '@/domain/Role'
+import { deleteRequest } from '@/services/http'
 
 async function fetchRoles(): Promise<void> {
   if (getRoles?.value?.length > 0) {
     return
   }
 
-  const roles = await get<Role[]>('/roles')
+  const fetchedRoles = await get<Role[]>('/roles')
 
-  setRoles(roles)
+  setRoles(fetchedRoles)
 }
 
-async function addRole(role: Role): Promise<void> {
-  await post<Role[]>('/roles', role)
+async function addRole(role: Role) {
   roles.value.push(role)
+  await post<Role[]>('/roles', role)
+}
+
+async function editRoleAction(role: Role): Promise<void> {
```

Comment:
```suggestion
async function editRole(role: Role): Promise<void> {
```
Can you rename this function?